_sources/Iteration/TraversalandtheforLoopByIndex.rst

```diff
@@ -27,29 +27,10 @@ allows us to loop through something and have an automatic counter.
        print(counter, item)
```

Comment:
I think we can make it clearer that we're using enumerate as a building block to help their understanding here. I'll do a proposed rewrite as a separate pull request.